blocks/table/table.css

```diff
@@ -183,6 +183,15 @@ main .table.text-table table tr td {
     flex-basis: calc(100% - 7.5em);
 }
 
+/* Blue theme */
+main .table.theme-blue table tbody tr td {
+    color: var(--secondary);
+}
+
+main .table.highlight-first-column table td:nth-child(1) {
```

Comment:
this style broke the Table having Merged Cell, let me try adding empty div with (display:none) to handle this.

<img width="526" alt="image" src="https://github.com/hlxsites/sunstar/assets/119420493/45d807b0-bf10-421b-a423-7c8608da843b">

<img width="782" alt="image" src="https://github.com/hlxsites/sunstar/assets/119420493/7187e3d5-bdf4-41c1-a51a-c88b61f222f9">

Reply:
Logged: https://github.com/hlxsites/sunstar/issues/407